migrations.config.ts

```diff
@@ -23,6 +23,14 @@ const DRIVER_MAP: Record<DbEngine, () => Promise<Options['driver']>> = {
 
 const isProduction = process.env.NODE_ENV === 'production';
 
+function resolveDriverOptions(): Dictionary {
+  if (process.env.SHLINK_DASHBOARD_DB_USE_ENCRYPTION === 'true') {
+    return { connection: { ssl: true } };
```

Comment:
I was looking at some docs and I'm wondering if something like this should be used so that connections to databases with self-signed certificates work as well https://github.com/mikro-orm/mikro-orm/issues/303#issuecomment-737342298

Reply:
Since you already have everything set up, could you test if that also works?